src/tests/multihost/alltests/test_kcm.py

```diff
@@ -52,3 +52,36 @@ def test_client_timeout(self, multihost, backupsssdconf):
                                                    " /var/log/sssd/"
                                                    "sssd_kcm.log")
         assert 'Terminated client' in grep_cmd.stdout_text
+
+    def test_kcm_check_socket_path(self, multihost, enable_kcm):
+        """
+        @Title: kcm: Test socket path when sssd-kcm is activated by systemd
```

Comment:
Use colon ":" before title instead of '@'
Leave a blank line and next line can add 
ticket: full link https://github.com/SSSD/sssd/issues/5406 (without a colon at the start)